specification/cognitiveservices/OpenAI.Inference/models/transcription.create.tsp

```diff
@@ -0,0 +1,95 @@
+import "@typespec/rest";
+import "@typespec/http";
+
+using TypeSpec.Rest;
+using TypeSpec.Http;
+
+namespace Azure.OpenAI;
+
+@doc("Transcription request.")
+model TranscriptionsOptions {
+
+    @doc("The audio file object to transcribe. Binary data.")
+    // how do we handle binary format for a member?
+    file: string;
+
+    @doc("An optional text to guide the model's style or continue a previous audio segment. The prompt should match the audio language.")
+    prompt?: string;
+
+    @doc("The format of the transcription output, in one of these options: json, text, srt, verbose_json, or vtt.")
+    @projectedName("json", "response_format")
+    responseFormat?: TranscriptionFormat = TranscriptionFormat.json;
+
+    @doc("The sampling temperature, between 0 and 1. Higher values like 0.8 will make the output more random, while lower values like 0.2 will make it more focused and deterministic. If set to 0, the model will use log probability to automatically increase the temperature until certain thresholds are hit.")
+    temperature?: float32 = 0;
+
+    @doc("The language of the input audio. Supplying the input language in ISO-639-1 format will improve accuracy and latency.")
+    language?: string;
+}
+
+@doc("Transcription format.")
+enum TranscriptionFormat {
+
+    @doc("JSON format.")
+    json : "json";
+
+    @doc("Text format.")
+    text : "text";
```

Comment:
I believe everything *except* `text` needs to be optional, and that'll be extra important in the likely case that we choose to hide the forked JSON/text response content types for VTT/STT (and instead just put the text into the appropriate JSON field).